.github/workflows/scanner-dev-vuln-update.yaml

```diff
@@ -107,6 +108,10 @@ jobs:
         fetch-depth: 0
         ref: ${{ github.event.pull_request.head.sha || github.ref }}
 
+    - uses: ./.github/actions/job-preamble
+      with:
+        gcp-account: ${{ secrets.GCP_SERVICE_ACCOUNT_STACKROX_CI }}
+
```

Comment:
The cleanup (`docker system prune...`) in the preamble wasn't enough for the release job, also had to also add: 
https://github.com/stackrox/stackrox/blob/a0c7efe935ef8ac0d4cc051402f68e16fe19f4e8/.github/workflows/scanner-release-vuln-update.yaml#L109-L111

May make sense to add that here as well (or to the preamble job itself if will not break other workflows)

Reply:
I updated this job to include that, but I also modified the release CI to call the preamble